frontend/packages/integration-tests-cypress/views/logs.ts

```diff
@@ -0,0 +1,29 @@
+export const logs = {
+  toggleOptions: () => {
+    // click configuration dropdown
+    cy.get('button[data-test="resource-log-options-toggle"]').click();
```

Comment:
i think we can also use that selector method here

```suggestion
    cy.byTestID('resource-log-options-toggle').click();
```